content/ConnectingToDatabase/ConnectingToDatabase.q.md

```diff
@@ -0,0 +1,673 @@
+# Connecting to Database
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    $host='localhost';
+    $db = 'MyDatabase';
+    $port = '5432';
+    $username = 'postgres';
+    $password = 'Admin01';
+    
+    $dsn = "pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password";
+
+    try{
```

Comment:
Please follow the PSR 2 coding guidelines for proper way of writing PHP codes. Please check this link. Thanks.

https://www.php-fig.org/psr/psr-2/